src/model.py

```diff
@@ -0,0 +1,23 @@
+class address_obj(object):
+    """
+    create address object initialized with
+    address, latitude, longitude and trash_days
+    """
+
+    def __init__(self,address,lat,lng,trash_days):
+        self.address = address
+        self.lat = lat
+        self.lng = lng
+        self.trash_days = trash_days
+    
+    def get_address(self):
+        return self.address
+
+    def get_lat(self):
+        return self.lat
+    
+    def get_lng(self):
+        return self.lng
+        
+    def get_trash_days(self):
```

Comment:
delete

Reply:
Once we have a list of objects, don't we then need methods to call the object's address, lat, lng and days? I think I'm failing to see the benefit of using objects in this case as opposed to a table. In order to select a specific set of addresses (for example, that fall on a certain trash day), don't we need to call each object's trash_days method?